demos/theme/theme-baseline.scss

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "../../packages/mdc-theme/color-palette";
+
+// TODO(acdvorak): Use theme mixin(s) instead
```

Comment:
don't put your name here...that implies you are going to do all this work very soon. While in reality it will probably be many people over several months.

Reply:
Done